persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/StudyMapper.xml

```diff
@@ -22,12 +22,27 @@
         </if>
     </sql>
 
+    <sql id="selectDetailed">
+        ,
+        COUNT(CASE WHEN sample_list.STABLE_ID = CONCAT(cancer_study.CANCER_STUDY_IDENTIFIER,'_sequenced') THEN 1 ELSE NULL END) AS sequencedSampleCount,
```

Comment:
Two concerns .. first, I still worry about performance here due to the additional joins, but I was persuaded that this should be manageable because we use similar joining logic elsewhere. second, I think we may be moving away from sample_list and sample_list_list ... in which case we will need a new implementation of this functionality. But for now this is ok.